demo/src/DownloadSVGButton.tsx

```diff
@@ -9,8 +9,7 @@ export default function DownloadSVGButton(props: DownloadSVGButtonPropTypes) {
 
     return (
         <Button
-            variant="contained"
-            color="primary"
```

Comment:
I'm not sure why you deleted this

Reply:
Hi Professor, we changed the variants of the Download SVG and Download JSON buttons so that they would look different from the Draw Diagram button (because we wanted the Draw Diagram button to stand out). We'd be happy to change it back, however!